src/Logging/Internal/OmexLoggerOptionsSetup.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
OmexLoggerOptionsSetup configures a OmexLoggingOption by using [ConfigurationBinder.Bind()](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.configurationbinder.bind) against an [IConfiguration](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.iconfiguration). OmexLoggerOptionsSetup class essentially binds a OmexLoggingOption instance with a section in the appsettings.json file. This is a crucial connection especially if we want to bind appsettings.json regarding our OmexLoggerProvider. Don't worry, it’s just plumbing.

You can decompile this [LoggerProviderOptions.RegisterProviderOptions](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.configuration.loggerprovideroptions.registerprovideroptions?view=net-8.0) to see the underlaying implementation which is similar to this change **but** for IOptionsMonitor. This change is to use IOptions only as we do not need hot config reloading.